modules/javafx.graphics/src/main/java/javafx/scene/Scene.java

```diff
@@ -1281,31 +1281,74 @@ void doCSSLayoutSyncForSnapshot(Node node) {
     // Shared method for Scene.snapshot and Node.snapshot. It is static because
     // we might be doing a Node snapshot with a null scene
     static WritableImage doSnapshot(Scene scene,
-            double x, double y, double w, double h,
-            Node root, BaseTransform transform, boolean depthBuffer,
-            Paint fill, Camera camera, WritableImage wimg) {
-
-        Toolkit tk = Toolkit.getToolkit();
-        Toolkit.ImageRenderingContext context = new Toolkit.ImageRenderingContext();
-
-        int xMin = (int)Math.floor(x);
-        int yMin = (int)Math.floor(y);
-        int xMax = (int)Math.ceil(x + w);
-        int yMax = (int)Math.ceil(y + h);
+                                    double x, double y, double w, double h,
+                                    Node root, BaseTransform transform, boolean depthBuffer,
+                                    Paint fill, Camera camera, WritableImage wimg) {
```

Comment:
Generally we avoid only formatting changes. It will be acceptable to change formatting of the code which is getting modified for the fix. Please do not include only formatting changes, which are not required for the fix.